src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -576,8 +577,7 @@ export default class DataTable extends Vue {
 
   _rowAccordionContent(accordionData: DataTableRowNestedContent, contentLevel: 'parent' | 'child') {
     if (accordionData.template) {
-      // eslint-disable-next-line
-      const template = this.$scopedSlots[accordionData.template]!(accordionData.payload);
+      const template = (this.$scopedSlots[accordionData.template] as NormalizedScopedSlot)(accordionData.payload);
```

Comment:
I don't think this really validates if the payload template/payload is actually provided. Can you test specifying template name for a row content but not providing `<template #testTemplate>` or payload and see how it behaves?

Reply:
Aye! It does seem to validate it as in both cases the result is an error thrown (this.$scopedSlots[accordionData.template] is not a function) and the data table not loading.

Reply:
I understand that using NormalizedScopedSlot removes the lining error that was showing, but it does not seem to be appropriate to be used here.